src/test/java/core/basesyntax/FileReaderServiceTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import core.basesyntax.service.FileReaderService;
+import core.basesyntax.service.impl.FileReaderServiceImpl;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderServiceTest {
+    private static List<String> excepted;
+    private static FileReaderService fileReaderService;
+    private static final File TEST_FILE = new File("src/test/resources/fileForFileReaderTest.csv");
+    private static final File NON_EXISTENT_FILE = new File("src/test/resources/bandera.csv");
+
+    @BeforeClass
+    public static void beforeClass() {
+        fileReaderService = new FileReaderServiceImpl();
+        excepted = new ArrayList<>();
+        excepted.add("type,fruit,quantity");
+        excepted.add("b,durian,100");
+        excepted.add("b,papaya,55");
+        excepted.add("p,durian,28");
+        excepted.add("s,papaya,45");
```

Comment:
move to Before()

in other words, let's initialize only service in @BeforeClass everywhere